### DISCUSSION TITLE
Unkown field in weekly budget

### DISCUSSION THREAD

**Discussion by sinux-l5d** [firefly-iii/firefly-iii#6458] [Firefly III]
Hi,

First, thanks for this amazing software: I've been using it for a month, and I'm starting to see the benefits.

So there is something I don't understand about weekly budgets: 
![image](https://user-images.githubusercontent.com/22861912/192000705-e569f95f-21b4-4af2-944a-9317aacce6d1.png)
_Alimentation (FR) = Feeding (EN)_

So it's 23th of September 2022. Each week, it sets a budget of 60€ for food. 

What is the last field which doesn't have a label in the "Budgeted" column?
In the "Budget" column, the last link is for the budget of the month, which seems to be 60€. But it's supposed to be at least 4*60€, isn't it?

So if someone could demystify this, I wound really appreciate :smile:

**Comment by JC5** [comment]
It only shows the active week, if I recall correctly.

**Reply by sinux-l5d** [reply]
Okey, but what about the monthly budget being at 60€? Shouldn't it be in proportion to the weekly budget?

**Reply by JC5** [reply]
It's a weekly budget, so it'll stay on a weekly schedule.

**Reply by sinux-l5d** [reply]
But I mean, I have this in the food budget (generated automatically): 
![Screenshot_20220923_190730](https://user-images.githubusercontent.com/22861912/192030225-1f2413c2-e7f1-4093-97a3-b467b4558b73.png)

Which leads to have a negative graph. Which is weird, as I respected my budget...
![Screenshot_20220923_190912](https://user-images.githubusercontent.com/22861912/192030470-51a319f6-6725-4bf0-af2f-6427ae6ad059.png)

**Reply by JC5** [reply]
You can have weekly and monthly budget amounts at the same time. This may be confusing, so I suggest using only period one at the time.

**Reply by sinux-l5d** [reply]
Could you explain a bit more on that? In the settings of this budget, I chose weekly. So no month budget should exists, but still there is one. So how is it connected?

**Reply by JC5** [reply]
Ah I see what you mean, sorry about that. I'll check it out.

**Reply by JC5** [reply]
Sorry, simple solution: Click the arrow next to the budget of 60 on the bottom of the list. Remove it. That should fix the issue.
